### PR TITLE
r-spatest try again

### PR DESCRIPTION
**Pull request by endrebak**
Testing

### CODE REVIEW

**Comment by endrebak** [comment]
Can this be merged please? Need it to add the new brilliant SAIGE algorithm to bioconda.

**Comment by bgruening** [comment]
@endrebak sorry for missing this PR. There are multiple things missing, could you please use https://github.com/bgruening/conda_r_skeleton_helper to create the recipe? It should take care of all the details. Thanks!

File: recipes/r-spatest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://CRAN.R-project.org/package=SPAtest
+  license: GPL (>= 2)
```
Comment (scopatz) [inline recipes/r-spatest/meta.yaml]: If it is GPLe the license needs to be distributed with the code.

Comment (endrebak) [reply recipes/r-spatest/meta.yaml]: Thanks.

**Comment by endrebak** [comment]
Now all I have done is:

1. Run conda_r_skeleton_helper
2. Changed the maintainer to me

Nothing else. Hoping this will work :)

**Comment by endrebak** [comment]
@bgruening Thanks for the help! Could you please merge? This is a required package for SAIGE which I will add to and maintain in bioconda (https://www.biorxiv.org/content/early/2017/11/01/212357)

**Comment by endrebak** [comment]
@bgruening @scopatz Do any of you have the opportunity to merge this? If not I will have to create an issue methinks :)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**